plugins/repository-hdfs/src/main/java/org/elasticsearch/repositories/hdfs/HdfsRepository.java

```diff
@@ -118,23 +118,33 @@ private static FileContext createContext(URI uri, Settings repositorySettings)
             cfg.set(entry.getKey(), entry.getValue());
         }
 
-        // create a hadoop user. if we want some auth, it must be done different anyway, and tested.
-        Subject subject;
+        // Initialize the UGI with the configuration.
+        UserGroupInformation.setConfiguration(cfg);
+
+        // Debugging
+        if (UserGroupInformation.isSecurityEnabled()) {
+            LOGGER.info("Hadoop Security Is [ENABLED]");
+        } else {
+            LOGGER.info("Hadoop Security is [DISABLED]");
+        }
+        UserGroupInformation.AuthenticationMethod method = SecurityUtil.getAuthenticationMethod(cfg);
+        LOGGER.info("Using Hadoop authentication method : [" + method + "]");
```

Comment:
This will make the logger usage checker unhappy, can you rewrite this to be parameterized?